.lintrunner.toml

```diff
@@ -579,3 +579,20 @@ init_command = [
     'black==22.3.0',
 ]
 is_formatter = true
+
+[[linter]]
+code = 'USORT'
```

Comment:
my impression was that usort and black would fight with one another depending on which one would run first, hence why stuff like [ufmt](https://ufmt.omnilib.dev/en/stable/) exists.

Can we do a similar thing? e.g. have a `PYFMT` linter that first runs usort, then runs black

Reply:
@joshuadeng, you've replaced `ufmt` with `lintrunner` in torchrec, could you please provide any insight into whether there is any interference between `black` and `usort` when called from `lintrunner`?

Reply:
I never implemented applying suggestions to the same file from two different linters: https://github.com/suo/lintrunner/blob/main/src/lib.rs#L55-L59

So it's probably better to combine them into a single linter, since I think quite frequently they will both suggest changes.